docs/Node_Health_Management.md

```diff
@@ -0,0 +1,262 @@
+#  **Managing Node Health**
+
+A node provides the runtime environments for containers. Each node in a Kubernetes cluster has the required services to be managed by the master. Nodes also have the required services to run pods, including the container runtime, a kubelet, and a service proxy.
+
+OpenShift Container Platform creates nodes from a cloud provider, physical systems, or virtual systems. Kubernetes interacts with node objects that are a representation of those nodes. The master uses the information from node objects to validate nodes with health checks. A node is ignored until it passes the health checks, and the master continues checking nodes until they are valid.
+We can manage nodes in our instance using the CLI. When we perform node management operations, the CLI interacts with node objects that are representations of actual node hosts. The master uses the information from node objects to validate nodes with health checks.
+
+You can review cluster node health status, resource consumption statistics, and node logs using the below commands. Additionally, you can query kubelet status on individual nodes.
+
+
+**1. Node reporting Not Ready**
```

Comment:
```suggestion
## 1. Node reporting NotReady
```

By using those headings, they start showing up on the left navigation (after `mkdocs` does its magic).